src/components/AssetInput/AssetInput.stories.tsx

```diff
@@ -117,40 +68,169 @@ PlaceholderLibraryOnly.argTypes = {
 export const Image = Template.bind({});
 
 Image.args = {
-    asset: {
-        source: "upload",
-        name: "foo",
-        size: 2000,
-        type: "image",
-        extension: "JPG",
-        src: "https://picsum.photos/200/300",
-    },
+    assets: [
+        {
+            source: "upload",
+            name: "foo",
+            size: 2000,
+            type: "image",
+            extension: "JPG",
+            src: "https://picsum.photos/200/300",
+        },
+    ],
     actions,
 };
 
 export const Audio = Template.bind({});
 
 Audio.args = {
-    asset: {
-        source: "library",
-        sourceName: "Foobar",
-        name: "foo",
-        extension: "MP3",
-        size: 2000,
-        type: "audio",
-    },
+    assets: [
+        {
+            source: "library",
+            sourceName: "Foobar",
+            name: "foo",
+            extension: "MP3",
+            size: 2000,
+            type: "audio",
+        },
+    ],
     actions,
 };
 
 export const Icon = Template.bind({});
 
 Icon.args = {
-    asset: {
-        source: "library",
-        sourceName: "Foobar",
-        name: "foo",
-        type: "icon",
-        icon: <IconIcons />,
-    },
+    assets: [
+        {
+            source: "library",
+            sourceName: "Foobar",
+            name: "foo",
+            type: "icon",
+            icon: <IconIcons />,
+        },
+    ],
     actions,
 };
+
+const TemplateMultiInput: Story<AssetProps> = (args) => {
+    return <AssetInput {...args} />;
+};
+
+export const TwoAssets = TemplateMultiInput.bind({});
+
+TwoAssets.args = {
```

Comment:
I'd disable the `size` and `isLoading` prop for the MultiAsset stories since they're not used

Reply:
You could create a const for the `argTypes` and reuse it since it's always the same :)